pyspecdata/load_files/__init__.py

```diff
@@ -221,6 +221,19 @@ def look_inside(inp_directory):
     return retval
 
 
+def register_proc_lookup(newdict):
+    """this updates the dictionary at
+    pyspecdata.load_files.postproc_lookup
+    this is equivalent to passing newdict to the lookup
+    keyword argument of `find_file`.
+
+    Remember that you only have to pass the keyword argument or call this
+    function once per script, and the effects are persistent!
+    """
+    postproc_lookup.update(newdict)
```

Comment:
here he's making the update a function whereas before it was just in the code always called